spec/models/ability_spec.rb

```diff
@@ -95,6 +95,51 @@
       it{ should_not be_able_to(:new, Registration.new(conference_id: conference_with_closed_registration.id))}
       it{ should_not be_able_to(:create, Registration.new(conference_id: conference_with_closed_registration.id))}
 
+      # Test for user can register or not
+      let(:registration_for_registering_user) { create(:registration, conference: create(:conference), user: user) }
```

Comment:
Perhaps it is easier to understand what we are testing with this, if it is called user_registration ?